observation_portal/common/test_data/doc_configdb.json

```diff
@@ -0,0 +1,2381 @@
+{
```

Comment:
Added this = right now it's just a copy of the configdb dump we currently have in the repo as a proof of concept. Thinking it'd be good to slim down and de-LCO-ify it a bit ;)

Reply:
Edit: added a slimmed-down, non LCO version